core/spec.md

```diff
@@ -4061,10 +4064,41 @@ and the following Resource level attributes:
 
 - Constraints:
   - REQUIRED.
-  - If present, it MUST be a case sensitive value from the model defined
```

Comment:
I wonder if the definitions above need to take into account "ancestor"? To us it's obvious what we mean, but when we only talk about "next oldest" I wonder if someone well read that to only mean "look at timestamps".

Reply:
Wouldn't the above sentence cover that?
> For `compatibility` strategies that require understanding the sequence in
  which Versions need to be compatible, the server MUST use the [`ancestor`]
  (#ancestor-attribute) to determine the sequence of Versions.

We are clearly saying that any sequencing should be based on the ancestor. Would it be more clear if we move that sentence down, right under the enumeration options?

Reply:
it's probably fine as-is. let's see if anyone else from the team raises a concern